src/reference/informative/informA2_4.rst

```diff
@@ -10,6 +10,7 @@
 
     .. code-block:: none
 
+      <!-- This is not valid! -->
       <model name="myInvalidCellML" has other stuff in the element information item >
```

Comment:
This is invalid xml, so already disallowed by 1.2.1.1

What 1.2.4 says is that you can't have this:
```
<model name="myInvalidCellML">Hello how are you</model>
```
The letters in there are "character information items" in XML jargon, and are perfectly valid XML, but not allowed in CellML

Reply:
Ah, ok.  I'll change my example (and it will render properly because it's valid XML! Woot!)

Reply:
ce51ca7